src/check-repo-levels.js

```diff
@@ -0,0 +1,27 @@
+import {getProjects, hgmoPath, scmLevel} from './util/projects';
+import child_process from 'child_process';
+
+module.exports.setup = (program) => {
+  return program
+    .command('check-repo-levels')
+    .description('check that the hg server and production-branches.json agree about repo levels. Must have an active Mozilla SSH key.');
+};
+
+module.exports.run = async (project, options) => {
```

Comment:
appears that this doesn't actually use any arguments, do we need to include them in the parameter list? Is it just to maintain a consistent method signature?